web/src/Sidebar.scss

```diff
@@ -53,12 +53,12 @@
   background-color: $color-white;
   color: $color-gray;
 }
-.resLink.has-warnings::after {
-  content: "⚠";
-  margin-top: -5px;
-  margin-left: $spacing-unit / 2;
-  font-size: 30px;
-  color: $color-yellow;
+.resLink.has-warnings::before {
```

Comment:
nbd for this PR but: do we want to distinguish "warnings" and "alerts" at all?

Reply:
Yeah the errors/warnings/alerts taxonomy needs to be more fleshed out. cc @yuindustries